### DISCUSSION TITLE
Update 6.15

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#293] [Release]
* add search bar expand/collapse animation toggle preference
* ignore keyboard close event when dialog open
* add dynamic calendar icon support
* refactor `asyncSetEntryIcon`
* ensure edit text is focused during pill animation
* rename preference screens `UI` and `Features`
* add `Options` screen to preferences
* rename "User interface" to "Theme" (see issue #271)

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.15'>Update 6.15</a>.</em>

**Comment by ildar** [comment]
Thank you for your work. I moved to the Android 12 and yet stayed with TBL!
👍

I searched and found that noone yet asked the now-popular "Fling up from
the Favorites" gesture. Please? or should I file an FR?

**Reply by TBog** [reply]
I'm not sure what you mean with _"Fling up from the Favorites"_

**Reply by ltguillaume** [reply]
I guess @ildar means the dock?

**Comment by ildar** [comment]
Like tap an icon in Favorites bar and pull up.
Many launchers show the app list like that.
